TYPO3-Solr/ext-solr
# Add support for Apache Solr 8.5

# What this pr does

* Add's the support for Apache Solr 8.5 and migrates the schema for EXT:solr 11

# How to test

Check if extension docker setup and extension is running properly with Apache Solr 8.5

Fixes: #2562
